qemu/tests/hotplug_mem.py

```diff
@@ -64,9 +65,33 @@ def unplug_memory(self, vm, target_mem):
             self.hotplug_memory(vm, target_mem)
         return super(MemoryHotplugSimple, self).unplug_memory(vm, target_mem)
 
+    def get_guest_node_mem(self, vm, node):
+        """
+        Get memory size of one node in guest
+        :param vm: VM object
+        :param node: node id
+        :return: memory size of the node
+        """
+        session = self.get_session(vm)
+        check_cmd = "numactl --hardware | awk -F: '/%s size/ {print $2}'" % node
```

Comment:
Is this only works for Linux?

Reply:
@vivianQizhu Yes, it only works for Linux.  So I added L118 to check the os type first. Currently we don't check windows guest node since there is no good tools working well in windows. Thanks!